model/host/hostutil.go

```diff
@@ -433,7 +433,22 @@ func (h *Host) GenerateUserDataProvisioningScript(ctx context.Context, settings
 			if err != nil {
 				return "", errors.Wrap(err, "constructing Jasper command to fetch task data")
 			}
-			postFetchClient += " && " + getTaskDataCmd
+			if !h.ProvisionOptions.UseOAuth {
```

Comment:
If we are using oauth, should we consider adding a file in the folder where the data is expected with an explanation of why the data is missing in case someone didn't know about this change? Maybe something like whyIsTheDataMissing.txt? :)

Reply:
I added a note file that gets cleaned up on fetch